src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -115,29 +115,25 @@ public void close() throws Exception {
   }
 
   public void changeFlightState() {
-    // Check if sensors are online
-    boolean coneSensorUp = timeOfFlightSensor.isSensor0Connected();
-    boolean cubeSensorUp = timeOfFlightSensor.isSensor1Connected();
-
-    // Log whether sensors are online
-    sensor0Up.setBoolean(coneSensorUp);
-    sensor1Up.setBoolean(cubeSensorUp);
-
     // Initialize both values to -1 and only change if sensors are up
     int coneDistance = -1; 
     int cubeDistance = -1;
 
     // Check if cone sensor is up
-    if (coneSensorUp) {
-      coneDistance = timeOfFlightSensor.getDistance0();
-      hasConeLog.log(coneDistance <= mmConeActivationThreshold);
-    }
+    coneDistance = timeOfFlightSensor.getDistance0();
+    hasConeLog.log(coneDistance <= mmConeActivationThreshold);
```

Comment:
If coneDistance is -1, this will incorrectly be true. Same with the cube distance